components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -865,6 +865,22 @@ public void removeOAuthApplicationData(String consumerKey) throws IdentityOAuthA
 
     }
 
+    /**
+     * Remove all OAuth consumer applications of a tenant.
+     *
+     * @param tenantId Id of the tenant
```

Comment:
Change this as follows:
@param tenantId of the tenant

Reply:
I have placed it there as, `tenantId` is the parameter name and *Id of the tenant* is the description. Hope that's fine.